indico/modules/events/abstracts/templates/display/call_for_abstracts.html

```diff
@@ -0,0 +1,80 @@
+{% extends 'layout/conference_page_base.html' %}
+
+{% block title %}
+    {%- trans %}Call for Abstracts{% endtrans -%}
+{% endblock %}
+
+{% macro render_timestamp(label, date, completed=false, alternative_text=none) %}
+    <li{% if completed %} class="completed"{% endif %}>
```

Comment:
`<li class="{{ 'completed' if completed }}">`

Having the jinja conditionals inside the attribute value usually plays along better with syntax highlighting in various editors.